R/Model-validity.R

```diff
@@ -79,7 +79,7 @@ v_model_logistic_normal_mix <- function(object) {
 
 #' @describeIn v_model_objects validates that `component` is a list with
 #'   valid `ModelParamsNormal` objects as well as `weights` are correct.
-v_model_logistic_normal_fixed_mix <- function(object) {
+v_model_logistic_normal_fixed_mix <- function(object) { # nolintr
```

Comment:
is there a threshold we can increase for the accepted name length? I thought so?

Reply:
Yes, there is, it is currently set to 30, and if I set 35 then it covers these cases, but I though that 30 is a good limit and actually we should avoid longer functions, except these few exceptions.

Reply:
hm ok as well... if it is really only less than 5 exceptions in whole package... otherwise either change names or increase threshold